roles/docker/tasks/main.yml

```diff
@@ -69,7 +69,7 @@
   when: (not ansible_os_family in ["CoreOS", "Container Linux by CoreOS"]) and (docker_package_info.pkgs|length > 0)
 
 - name: check minimum docker version for docker_dns mode. You need at least docker version >= 1.12 for resolvconf_mode=docker_dns
-  raw: docker version -f "{{ '{{' }}.Client.Version{{ '}}' }}"
+  command: "docker version -f '{{ '{{' }}.Client.Version{{ '}}' }}'"
```

Comment:
This was originally changed to raw because ansible 2.2.1 treats the registered output as a jinja2 expression